src/fairness/weighted_tree/test/load_accounting_db.cpp

```diff
@@ -0,0 +1,302 @@
+/*****************************************************************************\
+ *  Copyright (c) 2020 Lawrence Livermore National Security, LLC.  Produced at
+ *  the Lawrence Livermore National Laboratory (cf, AUTHORS, DISCLAIMER.LLNS).
+ *  LLNL-CODE-658032 All rights reserved.
+ *
+ *  This file is part of the Flux resource manager framework.
+ *  For details, see https://github.com/flux-framework.
+ *
+ *  This program is free software; you can redistribute it and/or modify it
+ *  under the terms of the GNU General Public License as published by the Free
+ *  Software Foundation; either version 2 of the license, or (at your option)
+ *  any later version.
+ *
+ *  Flux is distributed in the hope that it will be useful, but WITHOUT
+ *  ANY WARRANTY; without even the IMPLIED WARRANTY OF MERCHANTABILITY or
+ *  FITNESS FOR A PARTICULAR PURPOSE.  See the terms and conditions of the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License along
+ *  with this program; if not, write to the Free Software Foundation, Inc.,
+ *  59 Temple Place, Suite 330, Boston, MA 02111-1307 USA.
+ *  See also:  http://www.gnu.org/licenses/
+\*****************************************************************************/
+
+#include <cstdlib>
+#include <iostream>
+#include <sqlite3.h>
+#include <tuple>
+#include <vector>
+
+#include "src/fairness/weighted_tree/test/load_accounting_db.hpp"
+
+using namespace Flux::accounting;
+
+std::shared_ptr<weighted_tree_node_t> node = nullptr;
+std::shared_ptr<weighted_tree_node_t> user_node = nullptr;
+std::shared_ptr<weighted_tree_node_t> root_ptr = nullptr;
+
+std::shared_ptr<weighted_tree_node_t>
+get_sub_banks (sqlite3 *DB,
+               const std::string &bank_name,
+               std::shared_ptr<weighted_tree_node_t> parent_bank,
+               sqlite3_stmt *b_select_shares_stmt,
+               sqlite3_stmt *b_select_sub_banks_stmt,
+               sqlite3_stmt *b_select_associations_stmt) {
+    int rc = 0;
+    int leaf_bank_usage = 0;
+
+    // bind parameter to prepare SQL statement
+    rc = sqlite3_bind_text (b_select_shares_stmt,
+                            1,
+                            bank_name.c_str (),
+                            -1,
+                            NULL);
+    if (rc != SQLITE_OK) {
+        fprintf (stderr, "Failed to fetch data: %s\n", sqlite3_errmsg (DB));
```

Comment:
Since this is C++ code and `<iostream>` was already included, I would go with: 

```suggestion
        std::cerr << "Failed to fetch data: " << sqlite3_errmsg (DB);
```

Reply:
You will need `std::endl` at the end as well.